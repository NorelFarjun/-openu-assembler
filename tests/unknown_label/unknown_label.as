L1: lea L2, r1
lea L1, r1
