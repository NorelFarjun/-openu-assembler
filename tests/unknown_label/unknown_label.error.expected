address for label 'L2' not found in assembly file
Bad input file - not outputting
